src/databases/coursesDb.js

```diff
@@ -42,8 +78,8 @@ const createCourseCreator = ({
   creatorId,
 }) => trx
   .insert({
-    userId: creatorId,
-    courseId,
+    user_id: creatorId,
+    course_id: courseId,
```

Comment:
you could use the `snakecase` function instead of define this values in this way, but just a comment

Reply:
I'll change it!